spec/features/admin_logins_spec.rb

```diff
@@ -24,12 +24,26 @@
       visit admin_login_path
 
       within('.login') do
-        fill_in 'session[email]', with: "test@example.com"
-        fill_in 'session[password]', with: "password"
+        fill_in 'session[email]', with: 'user@example.com'
+        fill_in 'session[password]', with: 'password'
       end
       click_link_or_button('Login')
 
       expect(current_path).to eq(admin_login_path)
     end
+
+    it "doesn't allow non-admin users to login and view admin dashboard" do
+      user = User.create(email: 'user@example.com', password: 'password')
+      visit admin_login_path
+
+      within('.login') do
+        fill_in 'session[email]', with: user.email
+        fill_in 'session[password]', with: user.password
+      end
+      click_link_or_button('Login')
+
+      expect(current_path).to_not eq(admin_login_path)
```

Comment:
That's a fair point.  Given that we discussed not implementing users until the second sprint, I let them redirect to the admin_login_path.  Also, we haven't agreed upon the page where users who've successfully logged in will go.